types/types.proto

```diff
@@ -157,7 +157,35 @@ message ResponseBeginBlock{
 }
 
 message ResponseEndBlock{
-  repeated Validator diffs = 1;
+  repeated Validator changes = 1;
+  ConsensusParams consensus_param_changes = 2;
+}
+
+message ConsensusParams{
+    BlockSizeParams block_size_params = 1;
```

Comment:
Propose s/BlockSizeParams/BlockSize/g etc.
ConsensusParams.BlockSize.MaxBytes, not
ConsensusParams.BlockSizeParams.MaxBytes.

Otherwise, what about nested params?
ConsensusParams.BlockSizeParams.SomethingElseParams.Thing?? We don't want that.